flow360/component/flow360_params/solvers.py

```diff
@@ -179,6 +179,10 @@ class NavierStokesSolver(GenericFlowSolverSettings):
     def _get_field_order(cls) -> List[str]:
         return ["*", "linearSolver"]
 
+    @classmethod
+    def _get_optional_objects(cls) -> List[str]:
+        return ["linearSolver"]
```

Comment:
we can keep it as this in this PR. But I think we can create a generic method for it, right? Detect if something is optional and detect if it is base model